README.md

```diff
@@ -11,66 +11,122 @@ pages with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/).
+- Click on the pencil icon to edit the page. That will take you to the corresponding page in github.
 
-There click on the pencil icon to edit the page. That will take you to the corresponding page in github.
+<kbd>![Screenshot of a page on info.arxiv.org and the location of the edit icon](/non-info/02-help-info-page.png)</kbd>
 
-![gzip page](non-info/help-github1.png)
+- On the page you want to edit in github, click the pencil to edit the page contents right in github.
 
-There click "edit" and you can edit contents of the page in github. Once you
-have made changes you can click preview to see how it will look when completed
-There are slight differences between the markdown we use and what
-github uses.
+<kbd>![Screenshot of a page in Github and the location of the edit icon](/non-info/01-help-github1a.png)</kbd>
 
-![gzip page](non-info/help-github2.png)
+- After you have made changes, click preview to see how it will look when completed. 
+**Please note:** Github uses "Github flavored markdown" which is different from the markdown we use to create our documentation. There may be small variations in how the markdown appears due to these differences.
 
-Once you are happy with your changes you can save the by going to the bottom of
-the edit and "Commit changes". Commit is the term github and git use for
-saving. You should put a brief note in the first text area. You may add longer
-note in the larger text area if you have more to say about your changes. Then
-you can commit your changes to the branch `develop`.
+<kbd>![Screenshot of the location of the preview button on the edited page in Github](/non-info/03-help-preview-github.png)</kbd>
 
-The above step will not put your changes live on info.arxiv.org. At that point
-you have only saved the edit in github.
+- Once you are happy with your changes you can save by clicking on the blue "Commit changes" button in the upper right hand corner.
+Commit is the term github and git use for saving.
 
-If you have more changes that are grouped with this change make them in github
-and commit them.
+<kbd>![Screenshot of the location of the commit button on the edited page in Github](/non-info/04-help-commit-github.png)</kbd>
 
-### 2nd: Make a Pull Request from `develop` to `master`
-Once you have done all the changes you wanted you need to get your changes
-reviewed. This is done by "creating a PR." PR stands for pull request.
+- Add a brief note in the "Commit message" area summarizing your edit. You may add longer
+note in the `Extended description` area if you have more to say about your changes.
 
-To make a new PR click the "Pull Requests" tab on the top of a github page. Then
-click the green "New Pull Request." You want changes from the branch `develop` to get put
-into the branch called `master`.
+- Make sure you have your email associated with your commits.
 
-![gzip page](non-info/help-pr1.png)
+- Note that a "new branch for this request and start a pull request" will be selected.
```

Comment:
It is better practice to create a new branch should be done *before* making changes, but I guess if the user didn't do that, this could be a backstop.

Reply:
@cbf66 As far as I can tell, if someone is going to edit in place, there is not a way to create a branch before making changes. Your options are to commit directly to the develop branch or create a new branch. 
See the original instructions: https://github.com/arXiv/arxiv-docs/tree/master